CODE_OF_CONDUCT.md

```diff
@@ -1,7 +1,5 @@
 <!-- This file is automatically added by @npmcli/template-oss. Do not edit. -->
```

Comment:
When you see this comment in a file it means changing it will fail CI because it's auto-generated from template-oss.  In this case it's sourced from https://github.com/npm/template-oss/blob/main/lib/content/CODE_OF_CONDUCT-md.hbs